subworkflows/nf-core/gatk_paired_somatic_variant_calling/main.nf

```diff
@@ -0,0 +1,110 @@
+//
+// Run GATK mutect2 in tumour paired mode, getepileupsummaries, calculatecontamination, learnreadorientationmodel and filtermutectcalls
+//
+
+params.mutect2_options          = [:]
+params.learnorientation_options = [:]
+params.getpileup_options        = [suffix: '_tumour']
+params.getpileupnorm_options    = [suffix: '_normal']
+params.calccontam_options       = [:]
+params.filtercalls_options      = [suffix: '_filtered']
+
+include { GATK4_MUTECT2                   as MUTECT2 }                   from '../../../modules/gatk4/mutect2/main'                   addParams( options: params.mutect2_options )
+include { GATK4_LEARNREADORIENTATIONMODEL as LEARNREADORIENTATIONMODEL } from '../../../modules/gatk4/learnreadorientationmodel/main' addParams( options: params.learnorientation_options )
+include { GATK4_GETPILEUPSUMMARIES        as GETPILEUPSUMMARIES }        from '../../../modules/gatk4/getpileupsummaries/main'        addParams( options: params.getpileup_options )
+include { GATK4_GETPILEUPSUMMARIES        as GETPILEUPSUMMARIES_NORMAL}  from '../../../modules/gatk4/getpileupsummaries/main'        addParams( options: params.getpileupnorm_options )
```

Comment:
I'd specify the `GETPILEUPSUMMARIES` as `GETPILEUPSUMMARIES_TUMOR` as you did for the `GETPILEUPSUMMARIES_NORMAL`